src/preprocessing/passes/bool_to_bv.cpp

```diff
@@ -39,165 +39,159 @@ PreprocessingPassResult BoolToBV::applyInternal(
       options::preprocessStep());
 
   unsigned size = assertionsToPreprocess->size();
-  for (unsigned i = 0; i < size; ++i)
+
+  if (options::boolToBitvector() == BOOL_TO_BV_ALL)
+  {
+    for (unsigned i = 0; i < size; ++i)
```

Comment:
I'd like us to get into the habit of using a) the correct type size without implicitly down casting and b) fixed-width types wherever applicable. In this particular case, `size` should be `size_t` (see https://github.com/CVC4/CVC4/blob/master/src/preprocessing/assertion_pipeline.h#L39).

Reply:
Sounds good -- done with 68cd1a8524ba835196351c8eb9a84cd3f15b9df2